README.org

```diff
@@ -103,7 +107,20 @@ I would love to extend Emacs-bootstrap to support other programming languages as
 If you think you've got your configuration of your favorite programming language perfectly,
 please don't hesitate to submit a PR.
 
+<<<<<<< HEAD
```

Comment:
Really sorry I introduced a conflict. My bad. I'll make sure this doesn't happen again. I need to get around learning proper git workflow.